MIP_grid_label.json

```diff
@@ -0,0 +1,59 @@
+        "gr6":"regridded data reported on a grid other than the native grid and other than the preferred target grid",
+        "gr6a":"regridded data reported in the region of Antarctica on a grid other than the native grid and other than the preferred target grid",
+        "gr6g":"regridded data reported in the region of Greenland on a grid other than the native grid and other than the preferred target grid",
+        "gr6z":"regridded zonal mean data reported on a grid other than the native latitude grid and other than the preferred latitude target grid",
+        "gr7":"regridded data reported on a grid other than the native grid and other than the preferred target grid",
+        "gr7a":"regridded data reported in the region of Antarctica on a grid other than the native grid and other than the preferred target grid",
+        "gr7g":"regridded data reported in the region of Greenland on a grid other than the native grid and other than the preferred target grid",
+        "gr7z":"regridded zonal mean data reported on a grid other than the native latitude grid and other than the preferred latitude target grid",
+        "gr8":"regridded data reported on a grid other than the native grid and other than the preferred target grid",
+        "gr8a":"regridded data reported in the region of Antarctica on a grid other than the native grid and other than the preferred target grid",
+        "gr8g":"regridded data reported in the region of Greenland on a grid other than the native grid and other than the preferred target grid",
+        "gr8z":"regridded zonal mean data reported on a grid other than the native latitude grid and other than the preferred latitude target grid",
+        "gr9":"regridded data reported on a grid other than the native grid and other than the preferred target grid",
+        "gr9a":"regridded data reported in the region of Antarctica on a grid other than the native grid and other than the preferred target grid",
+        "gr9g":"regridded data reported in the region of Greenland on a grid other than the native grid and other than the preferred target grid",
+        "gr9z":"regridded zonal mean data reported on a grid other than the native latitude grid and other than the preferred latitude target grid",
+        "gra":"regridded data in the region of Antarctica reported on the data provider's preferred target grid",
+        "grg":"regridded data in the region of Greenland reported on the data provider's preferred target grid",
+        "grz":"regridded zonal mean data reported on the data provider's preferred latitude target grid"
+    },
+    "version_metadata":{
+        "CV_collection_modified":"Sun Nov 12 10:41:04 2023 -0000",
+        "CV_collection_version":"6.2.58.54",
+        "author":"Daniel Ellis <daniel.ellis@ext.esa.int>",
+        "source_type_CV_modified":"Sun Nov 12 10:41:04 2023 -0000",
+        "source_type_CV_note":"Creation of this file. ",
```

Comment:
It would be useful to capture the upstream info somewhere, by default these will be captured in this commit at least to start, so we can keep a trace of where this came from. Maybe "Replicating content from CMIP6_CVs upstream" would be a more useful identifier? Taking a quick peek at the upstreams, these were all amended to add descriptions, there was no real iteration of the content, just the format/descriptions, so not much provenance to capture in hindsight